README.md

```diff
@@ -1,8 +1,8 @@
 # codescene-kubernetes
 ## Install Codescene standalone on kubernetes
-These instructions have been tested on AWS but should also work for GKE and standard K8S. The only *caveat* is configuring Persistent Storage is environment-specific.
+These instructions have been tested on AWS and GKE and standard K8S and with Istio. The only *caveat* is configuring Persistent Storage is environment-specific.
 
-##Memory allocation
+### Memory allocation
 In some docker installation documentation, the recommendation was for `300Mi`. Codescene 3.0, however, running on K8S seems to work better with a higher memory allocation. I have found that
```

Comment:
In what docker documentation they recommend 300 MB and how is it relevant to CodeScene?
Depending on the projects you analyze, CodeScene may require much more (a few gigabytes?) memory than just 500 MB.
Also, `I have found` should probably be `We have found`.

Reply:
K8S allows you to define limits or resources. The first is a hard stop. The latter is a request which can, and usually does, go higher. I have defined resource requests not limits. If you prefer, you can change the lines to
        resources:
            requests:
              memory: "700Mi"
              cpu: "200m"